src/redisenterprise/azext_redisenterprise/manual/_help.py

```diff
@@ -0,0 +1,278 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['redisenterprise operation-status'] = """
+    type: group
+    short-summary: Manage Redis Enterprise operation status
+"""
+
+helps['redisenterprise operation-status show'] = """
+    type: command
+    short-summary: Get the status of an operation
+    examples:
+      - name: Get the status of an operation.
+        text: |-
+               az redisenterprise operation-status show --operation-id "testoperationid" --location "West US"
+"""
+
+helps['redisenterprise'] = """
+    type: group
+    short-summary: Manage dedicated Redis Enterprise caches for your Azure applications
+"""
+
+helps['redisenterprise list'] = """
+    type: command
+    short-summary: List Redis Enterprise caches
+    long-summary: List details about all caches within current Subscription or provided Resource Group
+    examples:
+      - name: List all Redis Enterprise caches in a resource group.
+        text: |-
+               az redisenterprise list --resource-group "rg1"
+      - name: List all Redis Enterprise caches within the current subscription.
+        text: |-
+               az redisenterprise list
+"""
+
+helps['redisenterprise show'] = """
+    type: command
+    short-summary: Get information about a Redis Enterprise cache
+    examples:
+      - name: Get information about a Redis Enterprise cache.
+        text: |-
+               az redisenterprise show --cluster-name "cache1" --resource-group "rg1"
+"""
+
+helps['redisenterprise create'] = """
+    type: command
+    short-summary: Create new Redis Enterprise cache instance
+    long-summary: Create or update an existing (overwrite/recreate, with potential downtime) cache cluster \
+with an associated database
+    parameters:
+      - name: --persistence
+        short-summary: "Persistence settings"
+        long-summary: |
+            Usage: --persistence aof-enabled=XX aof-frequency=XX rdb-enabled=XX rdb-frequency=XX
+
+            aof-enabled: Sets whether AOF is enabled.  Allowed values: true, false.
+            aof-frequency: Sets the frequency at which data is written to disk.  Allowed values: 1s, always.
+            rdb-enabled: Sets whether RDB is enabled.  Allowed values: true, false.
+            rdb-frequency: Sets the frequency at which a snapshot of the database is created.  Allowed values: \
+1h, 6h, 12h.
```

Comment:
Could you add flag to show which property is required?

Reply:
None of the persistence-related properties (the properties you highlighted here) are required.